airflow_clickhouse_plugin/operators/clickhouse_operator.py

```diff
@@ -1,7 +1,6 @@
-from typing import *
+from typing import Optional, Iterable, Dict, Any, Union
```

Comment:
Please do not change anything except from direct purpose of a PR. If you want to perform a refactoring, please make it part of another PR.